examples/grpc-bridge/client/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM grpc/python
+
+WORKDIR /client
+
+COPY requirements.txt /client/requirements.txt
+
+# Cache the dependencies
+RUN pip install -r /client/requirements.txt
+
+# Copy the sources, including the stubs
+COPY client.py /client/grpc-kv-client.py
+COPY kv /client/kv
+
+# https://stackoverflow.com/questions/1260792/import-a-file-from-a-subdirectory/1260813#1260813
+#RUN touch /client/kv/__init__.py
```

Comment:
id avoid commented out commands

Reply:
@snowp that's actually moved to a file... let me clean it up